lighty-modules/lighty-restconf-nb-community/src/test/java/io/lighty/modules/northbound/restconf/community/impl/tests/YangPatchTest.java

```diff
@@ -0,0 +1,116 @@
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import org.opendaylight.mdsal.common.api.LogicalDatastoreType;
+import org.opendaylight.restconf.common.context.InstanceIdentifierContext;
+import org.opendaylight.restconf.common.patch.PatchContext;
+import org.opendaylight.restconf.common.patch.PatchEntity;
+import org.opendaylight.restconf.common.patch.PatchStatusContext;
+import org.opendaylight.restconf.common.patch.PatchStatusEntity;
+import org.opendaylight.restconf.nb.rfc8040.rests.transactions.MdsalRestconfStrategy;
+import org.opendaylight.restconf.nb.rfc8040.rests.utils.PatchDataTransactionUtil;
+import org.opendaylight.yangtools.yang.common.QName;
+import org.opendaylight.yangtools.yang.common.Revision;
+import org.opendaylight.yangtools.yang.data.api.YangInstanceIdentifier;
+import org.opendaylight.yangtools.yang.data.api.YangInstanceIdentifier.NodeIdentifier;
+import org.opendaylight.yangtools.yang.data.api.YangInstanceIdentifier.NodeIdentifierWithPredicates;
+import org.opendaylight.yangtools.yang.data.api.schema.ContainerNode;
+import org.opendaylight.yangtools.yang.data.api.schema.DataContainerChild;
+import org.opendaylight.yangtools.yang.data.api.schema.LeafNode;
+import org.opendaylight.yangtools.yang.data.api.schema.MapEntryNode;
+import org.opendaylight.yangtools.yang.data.api.schema.SystemMapNode;
+import org.opendaylight.yangtools.yang.data.impl.schema.Builders;
+import org.opendaylight.yangtools.yang.data.impl.schema.builder.impl.ImmutableMapNodeBuilder;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class YangPatchTest extends CommunityRestConfTestBase {
+
+    private static final QName BASE_Q_NAME = QName.create("instance:identifier:patch:module",
+            "instance-identifier-patch-module", Revision.of("2015-11-21"));
+    private static final String EXAMPLE_LIST = "my-list1";
+    private static final String EXAMPLE_LIST_NAME = "name";
+    private static final NodeIdentifierWithPredicates NODE_WITH_KEY = NodeIdentifierWithPredicates.of(
+            QName.create(BASE_Q_NAME, EXAMPLE_LIST));
+    private static final String MY_LEAF_11 = "my-leaf11";
+    private static final String MY_LEAF_12 = "my-leaf12";
+    private static final String CONTAINER_ID = "patch-cont";
+
+    @Test
+    public void patchDataReplaceTest() throws ExecutionException, InterruptedException, TimeoutException {
+        Assert.assertNotNull(getLightyController());
+        Assert.assertNotNull(getCommunityRestConf());
+
+        final ContainerNode patchContainerNode = getContainerWithData();
+        final YangInstanceIdentifier targetNodeMerge = YangInstanceIdentifier.builder()
+                .node(QName.create(EXAMPLE_LIST, "patch-cont"))
```

Comment:
Use here CONTAINER_ID.

Reply:
CONTAINER_ID used instead of creating Qname